src/client/entity-editor/creator-data/creator-data.js

```diff
@@ -0,0 +1,82 @@
+/*
```

Comment:
How does this file differ from `src/client/entity-editor/creator-data.js`? The naming is a little problematic.